### ISSUE TITLE
"+ New" drop-down menu in Drive and Team Drive

### ISSUE THREAD

**Issue by davidbenque** [xwiki-labs/cryptpad#1191]
- should be coded as a list using HTML `<ul>` and `<li>` elements.
- currently all elements before "sheet" are not accessible via keyboard
- add Aria role for drop down menus? (to be confirmed)
